src/app/context/wallet/BTCWalletProvider.tsx

```diff
@@ -221,6 +234,12 @@ export const BTCWalletProvider = ({ children }: PropsWithChildren) => {
   return (
     <BTCWalletContext.Provider value={btcContextValue}>
       {children}
+      <FilterOrdinalsModal
+        open={filterOrdinalsModalOpen}
+        onClose={setFilterOrdinalsModalOpen}
+        shouldFilterOrdinals={shouldFilterOrdinals}
+        setShouldFilterOrdinals={setShouldFilterOrdinals}
+      />
```

Comment:
I don't think this modal relates to BTCWalletProvider. It can be placed anywhere inside our DApp